tflab/ops/compile_ops.py

```diff
@@ -1,18 +1,35 @@
 import os
 import sys
 import tensorflow as tf
+from distutils.version import StrictVersion as sv
 
-
+#TODO add check for old_gcc
 def compile_ops(old_gcc=True):
     start_dir = os.curdir
     abspath = os.path.abspath(__file__)
     dname = os.path.dirname(abspath)
     os.chdir(dname)
-    os.environ["TF_INC"] = tf.sysconfig.get_include()
-    if old_gcc:
-        fill = ""
+
+    files = "word2vec_ops.cc word2vec_kernels.cc -o word2vec_ops.so"
+    ABI_compat = "" if old_gcc else "-D_GLIBCXX_USE_CXX11_ABI=0"
+
+    if (sv(tf.__version__) >= "1.5.0"):
+        inc_flags = " ".join(tf.sysconfig.get_compile_flags())
+        lib_flags = " ".join(tf.sysconfig.get_link_flags())
+        # In 1.5.0, inc_flags sets the ABI flag to 1
+        if ("-D_GLIBCXX_USE_CXX11_ABI" in inc_flags):
```

Comment:
This may at some point break depending on how TF wants to manage `tf.sysconfig.get_compile_flags()`